WebService/ComparisonEngineUI/ComparisonEngineUI/Views/DrinksPage.xaml

```diff
@@ -0,0 +1,13 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://xamarin.com/schemas/2014/forms"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             x:Class="ComparisonEngineUI.Views.DrinksPage">
+    <ContentPage.Content>
+        <StackLayout>
+            <Label TextColor="Black" Text="Drink List:"
+                VerticalOptions="Center" 
+                HorizontalOptions="Center" />
+            <Button Text="Edit" Command="{Binding EditDrinkCommand}"/>
+        </StackLayout>
```

Comment:
We probably need some kind of scroll view to be able to fill this page with clickable buttons.